### ISSUE TITLE
Question about text spacing and overlapping behaviour

### ISSUE THREAD

**Issue by SpectraCoder** [microsoft/Win2D#781]
When I draw a text of multiple lines, and adjust the line spacing so that the lines overlap, parts of the text clip through the other parts. Is there a way to set this behaviour? Sort of like blend modes?

![image](https://user-images.githubusercontent.com/40742589/90156216-7c5f4600-dd8c-11ea-97bc-a59ac49097cf.png)

**Comment by shawnhar** [comment]
Yes, this behavior is determined by what blend mode and composite modes you have set.  It looks like you're doing something unusual there since you have an outline effect around the edge of the text in this screenshot?  Please share full details of how you're drawing it.

**Comment by SpectraCoder** [comment]
Thanks for your quick response! It is correct that there is an outline. That's drawn separately, and I forgot to turn it off for the screenshot. 

Anyway, here is a snippet of the code responsible for drawing it:

```
canvasTextLayout = new CanvasTextLayout(args.DrawingSession, currentSubtitle, subtitleFormatting, Parameters.FrameWidth, Parameters.FrameHeight);                 
                
CanvasGeometry subtitleGeometry = CanvasGeometry.CreateText(canvasTextLayout);
CanvasStrokeStyle canvasStrokeStyle = new CanvasStrokeStyle();
                
args.DrawingSession.FillGeometry(subtitleGeometry, SubtitleFontParameters.SubtitlePosition.X, SubtitleFontParameters.SubtitlePosition.Y, SubtitleFontParameters.FontColor);

if (CheckBoxIsStrokeEnabled.IsChecked == true)
{
     args.DrawingSession.DrawGeometry(subtitleGeometry, SubtitleFontParameters.SubtitlePosition.X, SubtitleFontParameters.SubtitlePosition.Y, SubtitleFontParameters.StrokeColor, SubtitleFontParameters.StrokeSize, canvasStrokeStyle);
}
```

Where can I set those blend and/or composite modes?

**Comment by shawnhar** [comment]
Ah, you're using geometry rather than drawing directly via Win2D text APIs.

I think this is behavior is because the OutlineTextRenderer implementation in CanvasGeometry.cpp doesn't provide any way to configure the D2D1_FILL_MODE setting, so you're getting the default D2D1_FILL_MODE_ALTERNATE which gives exactly the result you are seeing.

Short of changing that Win2D implementation, I think the only way to change the geometry intersection behavior would be to convert each line of text separately into its own CanvasGeometry, then merge those geometries together (via CanvasGeometry.CombineWith, which does let you configure the intersection mode).

**Comment by SpectraCoder** [comment]
That's a shame, but at least I now know why I couldn't find it in the documentation. Thanks for your time. I will try to think of a way to circumvent this in my code.
